src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -80,11 +80,21 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             HelpMessage = "The weight added to routes learned over BGP from this virtual network gateway")]
         public int PeerWeight { get; set; }
 
+        //[Parameter(
+        //    Mandatory = false,
+        //    ValueFromPipelineByPropertyName = true,
+        //    HelpMessage = "ActiveActive feature flag")]
+        //public bool ActiveActive { get; set; }
```

Comment:
Yes but these command lets are not yet released as in there are no customers who use these command lets yet. We will release the at Ignite. They were just part of last PS release but no customers using it yet.